cogs/abs_game.py

```diff
@@ -0,0 +1,495 @@
+URL_REVAN_ICON = 'https://cdn.discordapp.com/attachments/770579028624801802/914390225743151134/revan_icon.png'
+URL_REVAN_THUMBNAIL = 'https://cdn.discordapp.com/attachments/770579028624801802/914325770732707890/revan.png'
+URL_HK_ICON = 'https://cdn.discordapp.com/attachments/770579028624801802/915512461359214602/hk_icon.png'
+
+TEXT_INTRO_AUTHOR = 'Revan says...'
+TEXT_INTRO_TITLE = 'So, you want to be an ab caller?'
+TEXT_INTRO_SUBTITLE = '*My Revanites understand what you do not.*'
+TEXT_INTRO_PLAYERS_HEADER = '\n\n👥 \u200B __**PLAYERS**__'
+TEXT_INTRO_PROMPT = f'\n\n**Send any message in this channel if you\'d like to join.**'
+TEXT_INTRO_TIME_FORMAT = 'The game will begin in approximately {0} seconds.'
+TEXT_INTRO_FOOTER = 'Prepare yourselves - the game is about to begin.'
+
+TEXT_GAME_START_TITLE = 'The game has begun.'
+TEXT_GAME_START_SUBTITLE = '*Your interference will be your undoing!*'
+TEXT_GAME_START_DIRECTIONS = '\n\n**Aberrations will become active. Watch closely!** \u200B 👀' \
+                             '\n \u200B \u200B \u200B 📵 Don\'t click out of your Discord window.' \
+                             '\n \u200B \u200B \u200B 🚷 Don\'t type anything before you\'re told to.'
+
+TEXT_GAME_EMOJI_AB_ACTIVE = '🟣'
+TEXT_GAME_EMOJI_AB_BLANK = '⚫'
+TEXT_GAME_CALL_TITLE_FORMAT = 'Call the {0} aberration now. \u200B '
+TEXT_GAME_CALL_IDENTIFIERS = ['FIRST', 'SECOND', 'THIRD', 'FOURTH']
+TEXT_GAME_CALL_SUBTITLES = ['*For the galaxy to survive, you must fall!*', '*No! I won\'t be denied!*',
+                            '*I will never give in!*', '*You cannot stop the sacrifice!*']
+TEXT_GAME_PLAYERS_LABEL = '\n\n👥 **PLAYERS:** \u200B '
+TEXT_GAME_PLAYERS_NONE = 'None.'
+
+TEXT_GAME_DEATH_TITLE_SINGULAR = 'Someone has died.'
+TEXT_GAME_DEATH_TITLE_PLURAL_FORMAT = '{0} people have died.'
+TEXT_GAME_DEATH_SUBTITLE = '*You could have avoided this fate!*'
+TEXT_GAME_DEATH_LABEL = '\n\n☠️ **DEAD:** \u200B '
+TEXT_GAME_DISQUALIFIED_TITLE = 'Observation: Someone has started typing too early!'
+TEXT_GAME_DISQUALIFIED_SUBTITLE = '*Assessment: This meatbag will not provide optimal killing satisfaction.*' \
+                                  '\n*Assurance: I will end their pathetic life, master.*'
+
+TEXT_GAME_WON_TITLE_SINGULAR = 'We have a winner!'
+TEXT_GAME_WON_TITLE_PLURAL_FORMAT = 'We have {0} winners!'
+TEXT_GAME_WON_SUBTITLE = '*I can\'t believe you...*'
+TEXT_GAME_WON_LABEL = '\n\n🎉 **WINNERS:** \u200B '
+TEXT_GAME_LOST_TITLE = 'GAME OVER!'
+TEXT_GAME_LOST_SUBTITLE = '*You are all fools!*'
+TEXT_GAME_LOST_LABEL = '\n\n🪦 **LOSERS:** \u200B '
+
+
+class AbsGame(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.active_channel = None
+        self.active_channel_lock = Lock()
+
+    @commands.command()
+    async def abs(self, ctx):
+        # Only recognize this command in the #revan-abs-game channel for now.
+        if ctx.channel.id != 914637952825573377:
```

Comment:
I assume you'll make this configurable later, probably should add a TODO in the comments so you dont forget

Reply:
NODDERZ